### ISSUE TITLE
Add way to enable Firebase debug logging at runtime and build time.

### ISSUE THREAD

**Issue by tcurkovic-codecacao** [firebase/quickstart-unity#242]
Is there a way for the testers to to test analytics?
If I want to send iOS code to Archive (for ad hoc distribution) all the arguments are not passed. Is there a way to pass -FIRDebugEnabled argument so my QA team will be able to test analytics using Debug view in Firebase?

**Comment by stewartmiles** [comment]
Are you including https://github.com/firebase/firebase-ios-sdk/blob/6d6ed82d4200125a0642ffc513cb6fe4e3c1b5d3/Firebase/Core/Public/FIRLoggerLevel.h ?

**Comment by stewartmiles** [comment]
https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/importing_objective-c_into_swift

**Comment by tcurkovic-codecacao** [comment]
Thanks for the help. I have realised I do not need to add anything to the Bridging header file. I was just calling the FirebaseLoggerLevel enum. So I have made these changes to the AppDelegate.swift. Now I have lines
```
FirebaseApp.configure()
FirebaseConfiguration.shared.setLoggerLevel(FirebaseLoggerLevel.max)
```

The app now compiles. But unfortunately there are no events in the Firebase console. Do I need to add any other code? Am I using this thing wrong?
